src/main/java/com/triangledeterminer/controller/TriangleController.java

```diff
@@ -1,14 +1,50 @@
 package com.triangledeterminer.controller;
 
-import org.springframework.web.bind.annotation.RequestMapping;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.ResponseBody;
 import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.context.request.WebRequest;
+
+import com.triangledeterminer.error.ErrorHandler;
+import com.triangledeterminer.error.TriangleDeterminerException;
+import com.triangledeterminer.model.APIError;
+import com.triangledeterminer.model.ShapeDeterminer;
+import com.triangledeterminer.model.TraingleType;
+import com.triangledeterminer.model.Triangle;
+import com.triangledeterminer.validation.Validator;
 
 @RestController
 public class TriangleController {
-	
-	@RequestMapping("/")
-	String greet() {
-		return "Hello From Spring Boot !!!";
+
+	private static final Logger logger = LogManager.getLogger(TriangleController.class);
+	@Autowired
+	private ShapeDeterminer triangleDeterminer;
+	@Autowired
+	private Validator validator;
+
+	@PostMapping(value = "/triangle/type")
+	@ResponseBody
+	ResponseEntity<Object> determineTriangleType(@RequestBody Triangle triangle) throws TriangleDeterminerException {
+		logger.info("Input Triangle Side1 : {}, Side2 : {}, Side3 : {} ", triangle.getSide1(), triangle.getSide2(),
```

Comment:
If you want to log the entire object, then you should implement toString in the triangle class and not access individual fields in the controller

Reply:
Ok, I will take care in future.